deploy/035_deploy_debridge_facet.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers } from 'hardhat'
```

Comment:
If I understand correctly, hardhat is deprecated and we do not need a hardhat deploy script anymore.

Reply:
yeah, but we still have the `deploy` folder, so I added it.
when we remove the folder, it will be removed together ;)